README.md

```diff
@@ -59,19 +59,6 @@ npm run server:dev:hmr
 ```
 go to [http://0.0.0.0:3000](http://0.0.0.0:3000) or [http://localhost:3000](http://localhost:3000) in your browser
 
-# Nitrous Quickstart
```

Comment:
I agree and the same goes for everything else but I rather have a section at the bottom for these (Heroku, Nitrous)